csharp_dotnetcore/Using-Cards/Using_Cards/wwwroot/default.htm

```diff
@@ -0,0 +1,25 @@
+﻿<!DOCTYPE html>
```

Comment:
For all C# project, we will need to take the 'new' F5-run page that we plan with marketing.